compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/WindowExceptionHandlerFactory.desktop.kt

```diff
@@ -0,0 +1,37 @@
+package androidx.compose.ui.awt
+
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.ExperimentalComposeUiApi
+import java.awt.Window
+import java.awt.event.WindowEvent
+import javax.swing.JFrame
+import javax.swing.JOptionPane
+import javax.swing.SwingUtilities
+
+/**
+ * Default [WindowExceptionHandlerFactory], which will show standard error dialog, and close the window after that
+ */
+@ExperimentalComposeUiApi
+object DefaultWindowExceptionHandleFactory : WindowExceptionHandlerFactory {
+    override fun exceptionHandler(window: Window) = Thread.UncaughtExceptionHandler { _, throwable ->
+        SwingUtilities.invokeLater {
+            JOptionPane.showMessageDialog(window, "An error has occurred", "Error", JOptionPane.ERROR_MESSAGE)
```

Comment:
Better show throwable content and maybe even few frames of stacktrace.

Reply:
> Better show throwable content and maybe even few frames of stacktrace.

I am not sure about this. It will take time to make the right decision.

There are frameworks (Android, for example), which intentionally don't show the stacktrace, because:
- they can contain sensitivity information (like passwords without masking)
- it usually useless for the user (for advanced users it would be helpful, though)

Instead they have "Send report" button.

If users want to show stacktrace or want to make "Send report" button, they can implement it themselves (with custom exception handler). In the future, we can make a default "Send report" (as `SendReportWindowExceptionHandlerFactory(onReport: () -> Unit)`)